site/src/components/common/Card/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './Card';
```

Comment:
Why do we need default export here?
I suggest `export {default as Card} from './Card'`